examples/llama2/README.md

```diff
@@ -36,6 +36,14 @@ Supported languages are Python, TypeScript, JavaScript, Ruby, Julia, Rust, C++,
 
 Requirements file: [requirements-qlora.txt](requirements-qlora.txt)
 
+### Inference optimization using ONNX Runtime GenAI
+For using ONNX runtime GenAI to optimize, follow build and installation instructions [here](https://github.com/microsoft/onnxruntime-genai).
```

Comment:
better to add section about how to install genai?

Reply:
GenAI requires a build. It's not a pure python only module. It has C++ build step. I am afraid that any steps here will likely go out of sync since the library is still in active development. Moreover, the package is not published to pypi yet.